project/templates/admin/app_list.html

```diff
@@ -0,0 +1,40 @@
+{% load i18n %}
+
+{% if app_list %}
+  {% for app in app_list %}
+    <div class="app-{{ app.app_label }} module{% if app.app_url in request.path %} current-app{% endif %}">
+      <table>
+        <caption>
+          <a href="{{ app.app_url }}" class="section" title="{% blocktranslate with name=app.name %}Models in the {{ name }} application{% endblocktranslate %}">{{ app.name }}</a>
+        </caption>
+        {% for model in app.models %}
+          <tr class="model-{{ model.object_name|lower }}{% if model.admin_url in request.path %} current-model{% endif %}">
+            {% if model.admin_url %}
+              <th scope="row"><a href="{{ model.admin_url }}"{% if model.admin_url in request.path %} aria-current="page"{% endif %}>{{ forloop.counter }}.{{ model.name }}</a></th>
+            {% else %}
+              <th scope="row">{{ forloop.counter }}.{{ model.name }}</th>
+            {% endif %}
+
+            {% if model.add_url %}
+              <td><a href="{{ model.add_url }}" class="addlink">{% translate 'Add' %}</a></td>
+            {% else %}
+              <td></td>
+            {% endif %}
+
+            {% if model.admin_url and show_changelinks %}
+              {% if model.view_only %}
+                <td><a href="{{ model.admin_url }}" class="viewlink">{% translate 'View' %}</a></td>
+              {% else %}
+                <td><a href="{{ model.admin_url }}" class="changelink">{% translate 'Change' %}</a></td>
+              {% endif %}
+            {% elif show_changelinks %}
+              <td></td>
+            {% endif %}
+          </tr>
+        {% endfor %}
+      </table>
+    </div>
+  {% endfor %}
+{% else %}
+  <p>{% translate 'You don’t have permission to view or edit anything.' %}</p>
+{% endif %}
```

Comment:
Mình nghĩ việc đánh số chỉ là để quy định thứ tự thôi. Nếu bỏ đánh số đi mà mọi thứ vẫn được sắp xếp theo thứ tự cũ thì bỏ đánh số đi cũng được. Việc customize `app_list.html` chưa đem lại nhiều lợi ích và đang làm tăng độ phức tạp của code base, đi ngược lại với định hướng của giai đoạn hiện tại.

Tham khảo: [Định hướng của Gen 2](https://docs.google.com/document/d/1InhG0Pd2Bpt-lY7rZAc7TyznlWhOxNCocRpVSFX720Y/edit?usp=sharing)

Reply:
ok nếu số thứ tự ko phải là yêu cầu thì mình bỏ đi cũng đc. Thực ra `app_list.html` này là copy của django admin ra thôi, nên gần như không khác j cái mặc định, chỉ có thêm phần số thứ tự thôi. Có nhiều cái mình không rõ là do yêu cầu cần có hay chỉ đơn giản là nice to have nên mình ko dám thay đổi hết, mà chỉ thay đổi phương pháp 😓 .